Assets/Scripts/PlayerMovement.cs

```diff
@@ -1,37 +1,36 @@
 using System;
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
-
 public class PlayerMovement : MonoBehaviour
-{
-    [SerializeField] 
-    private int _speed = 3; 
-    Rigidbody2D rb2d;
-    // Start is called before the first frame update
+{ 
+    [SerializeField] private int _speed = 3;
+    private bool Direction = true;
+    private Rigidbody2D Rb2d = null;
```

Comment:
The convention for names should be `_rigidbody2D`, also add a space between fields and methods.